docs/plugins/plugin-development-guide/naming.rst

```diff
@@ -0,0 +1,59 @@
+Naming changes
+--------------
+
+``PluginSkeleton`` provides some default classes and configurations. However, they must have some default values and names that should be changed,
```

Comment:
Although the whole text is very well-written, I'm pretty sure that a comma after `changed` is unnecessary :D